test/net/FrameworkDialogs/FolderBrowser/FolderBrowserDialogSettingsTest.cs

```diff
@@ -9,6 +9,9 @@ namespace MvvmDialogs.FrameworkDialogs.FolderBrowser
     public class FolderBrowserDialogSettingsTest
     {
         [Test]
+#if NETCOREAPP3_0_OR_GREATER
+        [Ignore("Need to fix for .Net Core and later")]
+#endif
```

Comment:
@FantasticFiasco here is a test that exposed a few missing properties in .net core 3 and above. Skipping the test to not break the build, but I believe this could be made into a separate issue.